modules/migration-debugging-velero-admission-webhooks.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
```

Comment:
You can't put an assembly in the `modules` directory. And modules cannot include other modules.